sbpy/data/ephem.py

```diff
@@ -62,6 +62,16 @@ def from_horizons(cls, targetids, id_type='smallbody',
             Arguments that will be provided to
             `astroquery.jplhorizons.HorizonsClass.ephemerides`.
 
+        Notes
+        -----
+        * For detailed explanations of the queried fields, refer to
+          `astroquery.jplhorizons.HorizonsClass.ephemerides <https://astroquery.readthedocs.io/en/latest/api/astroquery.jplhorizons.HorizonsClass.html#astroquery.jplhorizons.HorizonsClass.ephemerides>`_ and the
+          `JPL Horizons documentation <https://ssd.jpl.nasa.gov/?horizons_doc>`_.
+        * By default, all properties are provided in the J2000.0 reference
+          system. Different settings can be chosen using
+          additional keyword arguments as used by
+          `astroquery.jplhorizons.HorizonsClass.ephemerides <https://astroquery.readthedocs.io/en/latest/api/astroquery.jplhorizons.HorizonsClass.html#astroquery.jplhorizons.HorizonsClass.ephemerides>`_.
```

Comment:
I would suggest to use intersphinx instead to link to API docs in other packages as the actual links may change even when the API is kept the same.

Reply:
You can read my mind! Intersphinx is exactly what I was looking for! Thanks!